doc/source/examples/incompressible-flow/2d-lid‐driven-cavity-flow/lid‐driven-cavity-flow.rst

```diff
@@ -0,0 +1,306 @@
+
+
+
+Parameter file
+--------------
+
+Lethe simulations are controlled by *parameter files* which possess the extension ``.prm``. This is the default text format of the ParameterHandler class of the deal.ii library from which Lethe derives. For more information on this class, we refer to the `deal.II documentation <https://www.dealii.org/current/doxygen/deal.II/classParameterHandler.html>`_. 
+
+Parameter files are made of subsections which describe a portion of the simulation (e.g. mesh generation, physical properties, simulation control). In parameter files, line starting with ``#`` are comments. Parameters are set using the following syntax:
+
+.. code-block:: text
+
+    set parameter name = value
+
+The syntax is flexible. Parameters do not need to be specified in a specific order, but only within the subsection in which they belong. For a full list of the parameters within Lethe, we refer to the parameter page.
+
+To set-up the lid-driven cavity case, we first need to establish the mesh used for the simulation.
+
+
+Mesh
+~~~~~
+
+The ``mesh`` subsection specifies the computational grid:
+
+.. code-block:: text
+
+    subsection mesh
+        set type                 = dealii
+        set grid type            = hyper_cube
+        set grid arguments       = 0 : 1 : true
+        set initial refinement   = 6
+    end
+
+The ``type`` specifies the mesh format used. At the moment, Lethe supports two mesh format: ``dealii`` and ``gmsh``. ``dealii`` meshes are in-situ generated meshes for simple geometries. The type of grid generated is specified by the ``grid type`` parameters and this grid is parametrized by it's ``grid arguments``. We refer to the documentation of the deal.ii `GridGenerator <https://www.dealii.org/current/doxygen/deal.II/namespaceGridGenerator.html>`_ for a detailed explanation of the available grids. 
+
+Since the lid-driven cavity problem domain is a square, we use the *hyper_cube* ``grid_type``. The arguments of this grid type are the position of the bottom left corner, the position of the top right corner and the option to colorize the boundaries in order to give each of them a unique ID. The IDs will be used to set the boundary conditions on specific part of the boundary of the domain. The ID given to each face were given in the graphical description of the case. If ``colorize`` option were set to false, all boundaries would have been given the ID ``0``.
+
+It is a bit surprising that the position of the bottom left and the top right corner are specified by a single value. Since the geometry is a square, the positions of the corner is specified using a single number, assuming that this identifies both the x and y value associated with that point. Other grid generators, such as the ``hyper_rectangle``, allow for more flexibility.
```

Comment:
```suggestion
It is a bit surprising that the position of the bottom left and the top right corner are specified by a single value. Since the geometry is a square, the position of the corner is specified using a single number, assuming that this identifies both the x and y value associated with that point. Other grid generators, such as the ``hyper_rectangle``, allow for more flexibility.
```